constants/social.js

```diff
@@ -76,16 +76,34 @@ export const SHARE_NETWORKS = [
 ];
 
 export const PODCASTS_PLATFORMS = [
+  {
+    id: 'applepodcasts',
+    label: 'Apple Podcasts',
+    link:
+      'https://podcasts.apple.com/by/podcast/wir-by-беларуская-і-сусветная-культура/id1511316106',
```

Comment:
```suggestion
      'https://podcasts.apple.com/by/podcast/id1511316106',
```

I think that will be enough